src/staging/src/styles/megaMenu/megaMenu.scss

```diff
@@ -0,0 +1,561 @@
+@import '../variables.scss';
+
+/* ---------------- HEADER ----------------- */
+
+
+/* TODO Move to global.scss? */
+
+.bc-parks-logo {
+  max-height: 50px;
+  max-width: 199px;
+}
+.bc-parks-wordmark {
+  height: 50px;
+  max-width: 199px;
+}
+a.btn.book-campsite-btn {
+  background-color: $colorWhite;
+  border: solid 1px transparent;
+  color: $colorBluePrimary;
+  font-size: 13px;
+  padding: 0px 30px;
+  line-height:40px;
+  height:40px;
+  border-radius:20px;
+  &:hover {
+    background-color: $colorWhite;
+    color: $colorBluePrimary;
+    text-decoration: underline;
+    outline: 1px solid $colorGoldPrimary;
+  }
+  @media screen and (max-width: $desktopBreakpoint) {
+    margin-right:50px;
+    padding: 0px 10px;
+    line-height:30px;
+    height:30px;
+    border-radius:15px;
+  }
+
+}
+
+@media screen and (min-width: $desktopBreakpoint) {
+
+    .navbar--header {
+        max-width: 1140px; 
+        margin: 0px auto;
+        padding: 0px 20px;
+        height: 100px;
+    }
+}
```

Comment:
Here and many other places, we should be using bootstrap classes - esp not pixels

Reply:
agreed. suggest we finish this effort in BRS-444